tests/installation/reconnect_s390.pm

```diff
@@ -40,6 +40,20 @@ sub run() {
         select_console('iucvconn');
     }
     else {
+        if (get_var('ENCRYPT')) {
+            my $password = $testapi::password;
+            my $svirt    = select_console('svirt');
+            my $name     = $svirt->name;
+            $svirt->suspend;
+            type_string "export pty=`virsh dumpxml $name | grep \"console type=\" | sed \"s/'/ /g\" | awk '{ print \$5 }'`\n";
+            type_string "echo \$pty\n";
+            $svirt->resume;
+
+            wait_serial("Please enter passphrase for disk.*", 100);
+            type_string "echo $password > \$pty\n";
+            wait_serial("Please enter passphrase for disk.*", 100);
+            type_string "echo $password > \$pty\n";
+        }
```

Comment:
well... the reconnect_s390 covers everything related to rebooting the machine
the password needs to be typed twice during the (re)boot process... 
We have now to options:
1. move this to the encrypted password module, but also move (and duplicate) the wait for the system being booted 
2. leave it like this and live with it